src/client.ts

```diff
@@ -660,6 +656,12 @@ export class StreamChat<
    * Revokes all tokens on application level issued before given time
    */
   async revokeTokens(before?: Date | string | null) {
+    if (before === undefined) {
+      throw new Error(
+        'Passing before parameter is mandatory for application level token revocation',
+      );
+    }
```

Comment:
Let's just remove `?` from function arguments

Reply:
we can remove `?` but we use the call as plain javascript call for ex - in chat-qa which causes the before param to be undefined.
I think we should have the check for safety.

Reply:
Well, even in JS linter will provide you with error message. And there's nothing bad in sending undefined anyway as the backend will respond with an error. Having `?` in parameters is misleading as one might think that it is possible to call it without an argument